docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
Ok, from some offline discussions, there is a counter argument for leaving out `step` and also dropping `start` and `end` (`since / until`): The node is only configured to keep 1-2 minutes of historical data to keep memory usage down, which limits the usefulness of those stats. Furthermore, clocks may be out of sync across nodes, so some additional client complexity is necessary to properly use start & end anyway.

Along the same lines, we should not be logging data frequently enough (except in some special circumstances) that `step` should matter much (e.g. if you want 20s intervals, just request the default 10s intervals, and drop every other sample).

I'm inclined to start with this in order to unblock this PR and simplify the implementation, since it seems to meet our current MVP use case. Remember that this API can grow after this PR - this is just an initial version.

/discuss :)